static/app/views/projectInstall/platform.tsx

```diff
@@ -140,6 +144,28 @@ export function ProjectInstallPlatform({
     });
   }, [organization, currentPlatform, project?.id]);
 
+  const redirectWithProjectSelection = useCallback(
+    to => {
+      if (!project?.id) {
+        return;
```

Comment:
is the project's id a string? I am wondering if we should use the function `defined` here

Reply:
If is a string. If it is an empty string it would also be bad. So this is a bit more defensive than `defined` I would say.

Reply:
in the types, we have `number` too, but if comes as a string it shall be fine